### PR TITLE
[3.9] no fatal error missing local timestamp agency state

### PR DESCRIPTION
**Pull request by kvahed**
### Scope & Purpose

*Do not fatal error on missing local timestamp in persisted log entries in agent*

- [X] :hankey: Bugfix
- [ ] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [ ] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [ ] Backport for 3.9: *(Please link PR)*
  - [ ] Backport for 3.8: *(Please link PR)*
  - [ ] Backport for 3.7: *(Please link PR)*

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket:
- [ ] Design document:

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,14 @@
 v3.9.2 (XXXX-XX-XX)
 -------------------
 
+* No good reason to fatal error in agency state, when local database entries
+  lack local timestamp (legacy). In that situation, we will record epoch
+  begin as local time.
+
+* Very verbose warning from failing to parse GEO JSON in search. Has lead to
+  Billions of log lines on deployed services.
+
```
Comment (jsteemann) [inline CHANGELOG:10]:
```suggestion
  billions of log lines on deployed services.
```

File: CHANGELOG [hunk 2]
```diff
@@ -1,6 +1,14 @@
 v3.9.2 (XXXX-XX-XX)
 -------------------
 
+* No good reason to fatal error in agency state, when local database entries
+  lack local timestamp (legacy). In that situation, we will record epoch
+  begin as local time.
+
+* Very verbose warning from failing to parse GEO JSON in search. Has lead to
```
[the remark this answers is not in the document]
Comment (kvahed) [reply CHANGELOG]: thx. done

[the remark this answers is not in the document]
Comment (jsteemann) [reply CHANGELOG]: The changes to the log levels are now still part of the PR. Only the CHANGELOG entry for it is gone now.

**Review by jsteemann — APPROVED** [review]
LGTM!

**Review by KVS85 — APPROVED** [review]
LGTM.

**Pull request merged**